test/app-luatest/console_debugger_session_test.lua

```diff
@@ -37,6 +33,55 @@ local function dbg_failed_range(v)
             format(v, 1, 0x7fffff00)
 end
 
+local verbose = os.getenv("DEBUG") or false
+
+local function debuglog(...)
```

Comment:
I usually use print for debugging without special options like `DEBUG` env var. luatest by default suppress all output from a test, but you always can enable verbose logging with `-c`. I propose to do the same here.

Reply:
Hmm, that's very good point, let me check whether we change timings here, once such massive logging (and it would be very, very verbose) were enabled. That might simplify debugging scenario for `luatest` case, yes.

Reply:
I've deleted conditionals depending on `verbose`, and now output `debuglog` to the screen unconditionally. By default `luatest` hides script output and we don't see debug log, otherwise, given `-c -v` options passed we see all relevant output we want to see usually:

```diff
--- a/test/app-luatest/console_debugger_session_test.lua
+++ b/test/app-luatest/console_debugger_session_test.lua
@@ -33,13 +33,8 @@ local function dbg_failed_range(v)
             format(v, 1, 0x7fffff00)
 end
 
-local verbose = os.getenv("DEBUG") or false
-
 local function debuglog(...)
-    if not verbose then
-        return
-    end
-    print(...)
+    print('--', ...)
 end
 
 --[[
@@ -78,7 +73,7 @@ local function dbg_script(content)
         end
     end
     f:close()
-    debuglog("temporary name:", tmpname)
+    debuglog("Temporary name:", tmpname)
     return tmpname
 end
 
@@ -273,7 +268,6 @@ local sequences = {
 
 local function run_debug_session(cmdline, sequence, tmpfile, header)
     local short_src = fio.basename(tmpfile)
-    debuglog('short_src:', short_src)
     --[[
         repeat multiple times to check all command aliases
     ]]
@@ -300,14 +294,14 @@ local function run_debug_session(cmdline, sequence, tmpfile, header)
                     cmd = cmd .. '\n'
                 end
                 fh:write(cmd)
-                debuglog('cmd: "'..trim(cmd)..'"')
+                debuglog('Execute command: "'..trim(cmd)..'"')
             end
 
             local result
             local clean_cmd = trim(cmd)
             -- there should be empty stderr - check it before stdout
             local errout = fh:read({ timeout = 0.05, stderr = true})
-            debuglog('errout:', trim(errout))
+            debuglog('stderr output: ', trim(errout))
             if first and errout then
                 -- we do not expect anything on stderr
                 -- with exception of initial debugger header
@@ -318,7 +312,7 @@ local function run_debug_session(cmdline, sequence, tmpfile, header)
             end
             repeat
                 result = trim(unescape(fh:read({ timeout = 0.5 })))
-                debuglog('result:', result)
+                debuglog('stdout output:', result)
             until result ~= '' or result ~= clean_cmd
             if expected ~= '' then
                 t.assert_str_contains(result, expected, false)
@@ -326,7 +320,7 @@ local function run_debug_session(cmdline, sequence, tmpfile, header)
         end
         fh:close()
     end
-    debuglog('delete ', tmpfile)
+    debuglog('Delete temporary file ', tmpfile)
     os.remove(tmpfile)
 end
 
```

```
12:19 $ luatest --shuffle none app-luatest/console_debugger_session_test.lua
Tarantool version is 3.0.0-entrypoint-57-g5a3dd66ec
......
Ran 6 tests in 17.786 seconds, 6 succeeded, 0 failed
✔ ~/datetime/tarantoolt/test [tsafin/debugger-5 L|✚ 2…48⚑ 12] 
12:21 $ luatest -v app-luatest/console_debugger_session_test.lua
Tarantool version is 3.0.0-entrypoint-57-g5a3dd66ec
Running with --shuffle group:6395
Started on Wed Apr  5 12:21:40 2023
    debug.sequence:"bpd_fail".test_debugger ... (1.399s) Ok
    debug.sequence:"bpd_ok".test_debugger ... (2.926s) Ok
    debug.sequence:"bps_fail".test_debugger ... (2.418s) Ok
    debug.sequence:"bps_ok".test_debugger ... (1.474s) Ok
    debug.sequence:"full".test_debugger ... (6.560s) Ok
    self-debug.test_debug_self_invoke ... (3.203s) Ok
=========================================================
Ran 6 tests in 17.984 seconds, 6 succeeded, 0 failed
✔ ~/datetime/tarantoolt/test [tsafin/debugger-5 L|✚ 2…48⚑ 12] 
12:21 $ luatest -c -v app-luatest/console_debugger_session_test.lua |head -100
Tarantool version is 3.0.0-entrypoint-57-g5a3dd66ec
--      Temporary name: /tmp/lua_2LyMO6
--      Temporary name: /tmp/lua_LfnKvq
--      Temporary name: /tmp/lua_XitJcK
--      Temporary name: /tmp/lua_vjiJT3
--      Temporary name: /tmp/lua_h5CXAn
--      Temporary name: /tmp/lua_BtfhiH
Running with --shuffle group:1016
Started on Wed Apr  5 12:22:08 2023
    debug.sequence:"bpd_fail".test_debugger ... 
--      stderr output: 
--      stdout output:  luadebug: Loaded for 3.0.0-entrypoint-57-g5a3dd66ec
break via /tmp/lua_h5CXAn => /tmp/lua_h5CXAn:1 in chunk at /tmp/lua_h5CXAn:0
   1=> local date = require 'datetime'
   2   local T = date.new{hour = 3, tzoffset = '+0300'}
   3   print(T)
   4   os.exit(0)
luadebug>
...
```